bikeshare_app/config.py

```diff
@@ -8,29 +9,39 @@ class BaseConfig(object):
     DEBUG = False
     TESTING = False
     MONGODB_SETTINGS = {'db': 'bikeshare'}
+    LOGGING_FORMAT = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+    LOGGING_LOCATION = 'bikeshare.log'
+    LOGGING_LEVEL = logging.INFO
 
 class TestingConfig(BaseConfig):
     """Configuration for running unit tests"""
     DEBUG = False
     TESTING = True
     MONGODB_SETTINGS = {'db': 'bikeshare-test'}
+    LOGGING_LEVEL = logging.DEBUG
 
 class DevelopmentConfig(BaseConfig):
     """The configuration that should be run during development"""
     DEBUG = True
     TESTING = True
     MONGODB_SETTINGS = {'db': 'bikeshare-test'}
+    LOGGING_LEVEL = logging.DEBUG
 
 CONFIG = {
-    "development": "bikeshare_app.config.DevelopmentConfig",
-    "testing": "bikeshare_app.config.TestingConfig",
-    "default": "bikeshare_app.config.BaseConfig"
+    'development': 'bikeshare_app.config.DevelopmentConfig',
+    'testing': 'bikeshare_app.config.TestingConfig',
+    'default': 'bikeshare_app.config.BaseConfig'
```

Comment:
I think we ought to pick one way and stick to it. I think always having the trailing comma is a better choice. I've added it.

We should add a note about that into the CONTRIBUTING.md